src/main/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpoint.java

```diff
@@ -48,7 +48,7 @@
     String methodName = "createNewCase";
     log.with("UPRN", requestBodyDTO.getUprn())
         .with("requestBody", requestBodyDTO)
-        .info("Entering POST {}", methodName);
+        .info("Entering POST for createNewCase");
```

Comment:
why? what was wrong with this prev? we now have a string literal created for single use?

Reply:
done